Library/ViewModels/ProjectDescriptionViewModel.swift

```diff
@@ -5,7 +5,7 @@ import WebKit
 
 public protocol ProjectDescriptionViewModelInputs {
   /// Call with the project given to the view.
-  func configureWith(value: Project)
+  func configureWith(value: ProjectPamphletMainCellData)
```

Comment:
why was this changed from project to data?

Reply:
`ProjectPamphletMainCellData` is a tuple of Project and RefTag. Just to make sure that RefTag is sent along when tracking the Project Page Viewed event in that ViewModel.